README.md

```diff
@@ -24,10 +24,99 @@ An example manuscript with documentation comments is below:
 ```yaml
 - repo_url: https://github.com/greenelab/deep-review  # URL to GitHub repository with Manuscript source code
   html_url: https://greenelab.github.io/deep-review/  # URL for Manubot HTML output (usually hosted by GitHub Pages)
-  thumbnail_url: https://url_for_image.png  # optional: square screenshot (showing part of html_url) used as manuscript thumbnail image.
+  thumbnail_url: https://url_for_image.png  # optional: square thumbnail (showing part of html_url) used as manuscript thumbnail image.
   preprint_citation: doi:10.1101/142760  # optional: Manubot-style citation for a preprint corresponding to the manuscript
   journal_citation: doi:10.1098/rsif.2017.0387  # optional: Manubot-style citation for a published article corresponding to the manuscript
 ```
 
 For more information on how to create citations for `preprint_citation` and `journal_citation`, see Manubot's citation-by-identifier [documentation here](https://github.com/manubot/rootstock/blob/master/USAGE.md#citations).
 Note that in the future, we plan to scan the source repository for a thumbnail image, such that `thumbnail_url` may be a temporary field.
+
+## Thumbnail Guidelines
+
+Follow these guidelines when creating your thumbnail to help us maintain a beautiful, consistent, and streamlined catalog.
+They are listed roughly in order of most strict to least strict.
+
+- **1000 x 1000 pixels**
+Provide an image exactly 1000 pixels wide by 1000 pixels high.
+Why do we ask for such a large image when the thumbnails of the papers appear so small in the catalog?
+1) Web browsers can use the extra pixels to display a cripser image when zoomed in (especially important on high dpi displays).
+2) The layout/design of the catalog is subject to change, which may show the thumbnails at larger sizes.
+3) In general, we want to future-proof the catalog against increasing image resolution standards, and for Manubot's own changing needs.
+
+- **PNG format**
+Provide your image as a `.png` file.
+This avoids unsightly artifacts produced by other common image formats while still keeping file size to a minimum.
```

Comment:
See https://github.com/manubot/rootstock/issues/267#issuecomment-529611462: there may sometimes be valid reasons for JPG or SVG... or is a common format most important here?

Reply:
Not jpeg. SVG maybe, but here it's better to be consistent. In the distant future when browses support svg favicons, maybe.